src/data/teamData.json

```diff
@@ -0,0 +1,47 @@
+[
```

Comment:
Great job on putting the data in a `data.js` file! This is a great thing to do for separating the data from the component!

However, it would be best if you can assign this array to a variable and export the variable so that we can have many arrays that we can export for all components inside this `data.js` file.

For example:
```
export const teamMembers = [{name:"Allan","position":"Frontend Developer",
     "src":"/assets/meetOurTeam/allan.jpg",
     "linkedin":"https://www.linkedin.com/in/allan-ahmed/"
    },{...
```

And then you can import it by its name like the following:
```
import {teamMembers} from "....
```